CHANGELOG.md

```diff
@@ -20,12 +20,11 @@
 - Added make integration command
 - Upgrade RabbitMQ Docker image to 3.13
 - Updated README with steps on running the processes locally
-
+- Created local.yaml file to setup environment variables when running processes locally
```

Comment:
Awesome!

Reply:
Let's make an update here to create a new set of changes and bump the version!

Reply:
Sure makes sense